test/unit/extensions.js

```diff
@@ -152,6 +152,25 @@
 		strictEqual( $( '#el' ).data( "somekey" ), undefined, 'should be undefined on an nonempty result set with a key passed in.' );
 	});
 
+	test( '`.data` does not alias to `data-` attributes', function() {
+		expect( 3 );
+		var $fixture = shoestring( '#qunit-fixture' ),
+			$el;
+
+		$fixture.html( '<div id="el" data-attr1 data-attr2="test"></div>' );
```

Comment:
Hm, I like keeping the markup for the test close to the test code. Are you set on changing this?